lib/ansible/plugins/callback/actionable.py

```diff
@@ -32,6 +32,7 @@ def __init__(self):
         self.super_ref.__init__()
         self.last_task = None
         self.shown_title = False
+        self.diff = False
```

Comment:
Not entirely sure, what you mean by that :confused: 

I added the `self.diff` locally to override handling of diffs in this particular callback.

Are you proposing an entirely different and better fix?